ttnn/tutorials/005.ipynb

```diff
@@ -612,7 +612,7 @@
     "cd $TT_METAL_HOME\n",
     "unset SILENT\n",
     "make nuke\n",
```

Comment:
Out side of the scope but is this nuke still relavant?

Reply:
no idea, since its out of scope, someone who cares about that ipynb can fix it.

Reply:
Haha this is real documentation too, oh boy